Cargo.toml

```diff
@@ -33,7 +33,7 @@ ariadne = "0.1.2"
 pom = "3.2"
 nom = "7.1"
 serde_json = { version = "1.0", features = ["preserve_order"] }
-criterion = "0.4.0"
+criterion = { version = "0.4.0", features = ["html_reports"] }
```

Comment:
I can revert this, it was accidentally left in, though also probably doesn't hurt

Reply:
It's testing only and IIRC it used to be enabled in criterion by default anyway, so I think it's fine.

Reply:
Yeah, if you use the `.3` version, it gives some message along the lines of 'html_reports is becoming non-default, use it explicitly if you want it to keep working'. I added it in while playing with the benchmarks to get a better idea about the collected results.